Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -248,29 +263,33 @@ extension RouteController {
         public static let rawLocationKey: NotificationUserInfoKey = .init(rawValue: "rawLocation")
         
         /**
-         A key in the user info dictionary of a `Notification.Name.routeControllerDidFailToReroute` notification. The corresponding value is an `NSError` object indicating why `RouteController` was unable to calculate a new route.
+         A key in the user info dictionary of a `Notification.Name.currentRoadNameDidChange` notification. The corresponding value is a `NSString` object representing the current road name.
          */
-        public static let routingErrorKey: NotificationUserInfoKey = .init(rawValue: "error")
+        public static let roadNameKey: NotificationUserInfoKey = .init(rawValue: "roadName")
+        
+        // MARK: Marking Rerouting
```

Comment:
For consistency with the next mark:

```suggestion
        // MARK: Monitoring Rerouting
```